release-notes/opensearch-release-notes-1.0.1.md

```diff
@@ -0,0 +1,95 @@
+# OpenSearch and Dashboards 1.0.1 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.1 includes the following OpenSearch bug fixes, documentation and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0.md).
```

Comment:
These should be `1.0.1.md`, no?

Reply:
We are not changing the -min version so we are not generating the new release notes for OpenSearch